src/sentry/migrations/0477_backfill_organization_mappings_via_outbox.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-06-01 18:56
+
+from django.db import migrations
+
+from sentry.models import OutboxCategory, OutboxScope
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_org_mapping_via_outbox(apps, schema_editor):
+    Organization = apps.get_model("sentry", "Organization")
+    RegionOutbox = apps.get_model("sentry", "RegionOutbox")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.all()):
+        RegionOutbox(
+            shard_scope=OutboxScope.ORGANIZATION_SCOPE,
+            shard_identifier=org.id,
+            category=OutboxCategory.ORGANIZATION_UPDATE,
+            object_identifier=org.id,
+        ).save()
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = True
```

Comment:
I'm not entirely sure this is necessary? But I've heard that backfills typically require us to mark them as dangerous so I didn't want to break this precedence.